contentcuration/contentcuration/frontend/shared/views/channel/ChannelDetailsModal.vue

```diff
@@ -7,34 +7,51 @@
     <template #header>
       <span class="notranslate">{{ channel.name }}</span>
     </template>
+    <template v-if="adminMode" #tabs>
```

Comment:
I think eventually it might make sense to break out the 'info' tab into a component and just have an admin specific modal that uses the info component and the sharing component - just the 'adminMode' gave me a lilttle pause, as adding additional mode props to components is often a sign we are trying to make one component do too much work!

Reply:
Now that you mention it, I might be able to just use the Details component for its own modal